src/PosUi/itemSelection/index.js

```diff
@@ -30,7 +30,9 @@ const mapStateToProps = (state) => {
 
   const firstPredictionLabel = allPredictions?.[0]?.dataList[0];
 
-  const unknownItem = firstPredictionLabel === OTHER_FRUIT_LABEL;
+  const unknownItem = [OTHER_FRUIT_LABEL, undefined].includes(
+    firstPredictionLabel
+  );
```

Comment:
Maybe I wrong, but it looks like in case that predictor returns an empty list, this code is unreachable. It will return on line 20.

Reply:
you are absolutely right - I am fixing it